jekyll/_cci2/docker-to-machine.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: classic-docs
+title: "Migrating from Docker to Machine executor"
+short-title: "Migrate from Docker to Machine"
+description: "Migrate from Docker to Machine executors"
+categories: [containerization, configuration]
+order: 20
+---
+
+This document contains some general guidelines and considerations to make when
+moving from the Docker executor to machine, or vice versa.
+
+* TOC
+{:toc}
+
+## Overview
+{:.no_toc}
+
+Occiasonally, the Docker executor isn't quite the right fit for your builds - 
+this can include a lack of memory or requiring more dedicated CPU  power. Moving
```

Comment:
```suggestion
Reasons for choosing an alternative executor could include requiring more memory or more dedicated CPU  power. Moving
```

Reply:
The formatting is going to go all wrong here because the paragraph is split across multiple lines.